core/controllers/base.py

```diff
@@ -50,14 +49,6 @@
     'oppia_csrf_secret', {'type': 'unicode'},
     'Text used to encrypt CSRF tokens.', DEFAULT_CSRF_SECRET)
 
-BEFORE_END_HEAD_TAG_HOOK = config_domain.ConfigProperty(
```

Comment:
I believe we maintain somewhere (config_domain.py?) a list of deprecated config properties; we should add this one here, too.

Reply:
@Jamesjay4199 Make sure to respond to all comments, please.

Reply:
Yes, done